services/121-service/test/visa-card/__snapshots__/export-report.test.ts.snap

```diff
@@ -7,7 +7,7 @@ exports[`Export Visa debit card report should succesfully generate a report of a
     "cardStatus121": "Active",
     "explanation": "",
     "isCurrentWallet": true,
-    "lastUsedDate": null,
+    "lastUsedDate": "2025-02-07T00:00:00.000Z",
```

Comment:
I looked through the test code a bit, but it did not become obvious to me that lastUsedDate will always result in 7 Feb 2025 in the future. Does it? Or is this not how snapshots work?

Reply:
Ah good point